cleanup/cleanup.sh

```diff
@@ -1,3 +1,111 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-echo "Plugin called."
+IMAGE_NAME=$1
+
+set -e -o pipefail -u
+
+GIT_COMMIT_LIMIT=${KUBECTL_PLUGINS_LOCAL_FLAG_GIT_COMMIT_LIMIT}
+GIT_REPO_PATH=${KUBECTL_PLUGINS_LOCAL_FLAG_GIT_REPO_PATH}
+FORCE=${KUBECTL_PLUGINS_LOCAL_FLAG_FORCE}
+KEEP=${KUBECTL_PLUGINS_LOCAL_FLAG_KEEP}
+NAMESPACE=${KUBECTL_PLUGINS_CURRENT_NAMESPACE}
+oc="${KUBECTL_PLUGINS_CALLER} -n ${NAMESPACE}"
+
+get_commit_list() {
+    # all commits except HEAD in the Git repository
+    git -C ${GIT_REPO_PATH} rev-list --max-count ${GIT_COMMIT_LIMIT} --author-date-order HEAD~
```

Comment:
Double quote `${VARIABLES}` to prevent globbing and word splitting (here and everywhere in the code)